packages/@lwc/engine-dom/src/renderer/index.ts

```diff
@@ -193,6 +193,10 @@ function getTagName(elm: Element): string {
     return elm.tagName;
 }
 
+function attachInternals(elm: HTMLElement): ElementInternals {
+    return HTMLElement.prototype.attachInternals.call(elm);
```

Comment:
Call from the `prototype` because we're disabling access to the `attachInternals` method outside of it, ie.

```js
elm.attachInternals() // method doesn't exist on the element
```

Reply:
Can we add a comment about this?

Reply:
@nolanlawson comments [added](https://github.com/salesforce/lwc/pull/3670/files#diff-3c93f84cc4435bd58eb5d19d62dc1db90142581a55b2b4c6c4ccf3247e22976dR196-R201).